client/layout/index.jsx

```diff
@@ -131,6 +135,9 @@ Layout = React.createClass( {
 					<WelcomeMessage welcomeSite={ newestSite } />
 				</Welcome>
 				<TranslatorInvitation isVisible={ showInvitation } />
+				<PreferencesData>
+					<PollInvitation isVisible={ ! disablePollInvitation } section={ this.props.section } />
```

Comment:
More we add here, more I wish these welcome prompts were split off into a separate component.